tests/test_manipulation.py

```diff
@@ -332,6 +332,14 @@ def test_no_copy(self):
         assert_array_equal(b, a)
 
 
+class TestBroadcast:
+    @pytest.mark.parametrize("shape", [(1, 1), (0, 1)])
```

Comment:
I guess it needs to be updated like:
```suggestion
    @pytest.mark.parametrize(
        "shape",
        [
            [(1,), (3,)],
            [(1, 3), (3, 3)],
            [(3, 1), (3, 3)],
            [(1, 3), (3, 1)],
            [(1, 1), (3, 3)],
            [(1, 1), (1, 3)],
            [(1, 1), (3, 1)],
            [(1, 0), (0, 0)],
            [(0, 1), (0, 0)],
            [(1, 0), (0, 1)],
            [(1, 1), (0, 0)],
            [(1, 1), (1, 0)],
            [(1, 1), (0, 1)],
        ],
    )
```